docs/additional-features/chainbridge/definitions.md

```diff
@@ -0,0 +1,204 @@
+  --bridge "[BRIDGE_CONTRACT_ADDRESS]"
+
+# Unpause
+$ cb-sol-cli admin unpause \
+  --url [JSON_RPC_URL] \
+  --privateKey [ADMIN_ACCOUNT_PRIVATE_KEY] \
+  --bridge "[BRIDGE_CONTRACT_ADDRESS]"
+```
+
+### Change Fee
+
+Change the fee which will be paid to Bridge Contract
+
+```bash
+# Change fee for execution
+$ cb-sol-cli admin set-fee \
+  --url [JSON_RPC_URL] \
+  --privateKey [ADMIN_ACCOUNT_PRIVATE_KEY] \
+  --bridge "[BRIDGE_CONTRACT_ADDRESS]" \
+  --fee [FEE_IN_WEI]
+```
+
+### Add/Remove a relayer
+
+Add an account as a new relayer or remove an account from relayers
+
+```bash
+# Add relayer
+$ cb-sol-cli admin add-relayer \
+  --url [JSON_RPC_URL] \
+  --privateKey [ADMIN_ACCOUNT_PRIVATE_KEY] \
+  --bridge "[BRIDGE_CONTRACT_ADDRESS]" \
+  --relayer "[NEW_RELAYER_ADDRESS]"
+  
+# Remove relayer
+$ cb-sol-cli admin remove-relayer \
+  --url [JSON_RPC_URL] \
+  --privateKey [ADMIN_ACCOUNT_PRIVATE_KEY] \
+  --bridge "[BRIDGE_CONTRACT_ADDRESS]" \
+  --relayer "[RELAYER_ADDRESS]"
+```
+
+### Change relayer threshold
+
+Change the number of votes required for a proposal execution
+
+```bash
+# Remove relayer
+$ cb-sol-cli admin set-threshold \
+  --url [JSON_RPC_URL] \
+  --privateKey [ADMIN_ACCOUNT_PRIVATE_KEY] \
+  --bridge "[BRIDGE_CONTRACT_ADDRESS]" \
+  --threshold [THRESHOLD]
+```
+:::
+
+## Chain ID
+
+The Chainbridge `chainId` is an arbitrary value used in the bridge for differentiating between the blockchain networks, and it has to be in the range of uint8. To not be confused with the chain ID of the network, they are not the same thing. This value needs to be unique, but it doesn't have to be the same as the ID of the network.
+
+In this example, we set  `99` in `chainId`, because the chain ID of the Mumbai testnet is `80001`, which cannot be represented with a uint8.
+
+## Resource ID
+
+A resource ID is a unique 32-bytes value in a cross-chain environment, associated with a certain asset (resource) that is being transferred between networks.
+
+The resource ID is arbitrary, but, as a convention, usually the last byte contains the chain ID of the source chain (the network from which this asset originated from). 
+
+## JSON-RPC URL for Polygon PoS
+
+For this guide, we’ll use https://rpc-mumbai.matic.today, a public JSON-RPC URL provided by Polygon, which may have traffic or rate-limits. This will be used only to connect with the Polygon Mumbai testnet. We advise you to obtain your JSON-RPC URL by an external service like Infura because deploying contracts will send many queries/requests to the JSON-RPC.
+
+## Ways of processing the transfer of tokens
+When transferring ERC20 tokens between chains, they can be processed in two different modes:
+
+### Lock/release mode
+Source chain: The tokens you are sending will be locked in the Handler Contract.
+Destination chain: The same amount of tokens as you sent in the source chain would be unlocked and transferred from the Handler contract to the recipient account in the destination chain.
```

Comment:
When you open the doc there is not new line

Reply:
fixed 59249ed5d04a074ba6f611f1b8752776bbe7802b